tests/pytest/oauth/test_views.py

```diff
@@ -161,38 +158,12 @@ def test_cancel_no_session_flow(client):
 
 
 @pytest.mark.django_db
-def test_logout_no_session_flow(app_request):
-    result = logout(app_request)
-
-    assert result.status_code == 200
-    assert result.template_name == TEMPLATE_USER_ERROR
-
-
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification")
-def test_logout(app_request, mocker, mocked_oauth_client_or_error_redirect__client, mocked_view_analytics_module):
-    # logout internally calls deauthorize_redirect
-    # this mocks that function and a success response
-    # and returns a spy object we can use to validate calls
-    message = "logout successful"
-    mocked_oauth_client = mocked_oauth_client_or_error_redirect__client.return_value
-    mocked_redirect = mocker.patch("benefits.oauth.views.redirects.deauthorize_redirect", return_value=HttpResponse(message))
-
-    oauth_token_authorized = True
-    session.update(app_request, logged_in=oauth_token_authorized)
-    assert session.logged_in(app_request) == oauth_token_authorized
-
-    result = logout(app_request)
-
-    mocked_redirect.assert_called_with(app_request, mocked_oauth_client, "https://testserver/oauth/post_logout")
-    mocked_view_analytics_module.started_sign_out.assert_called_once()
-    assert result.status_code == 200
-    assert message in str(result.content)
+def test_logout_no_session_flow(client):
+    path = reverse(routes.OAUTH_LOGOUT)
+    response = client.get(path)
 
-    assert not session.logged_in(app_request)
-    assert session.enrollment_token(app_request) is False
-    assert session.logged_in(app_request) is False
-    assert OAuthSession(app_request).claims_result == ClaimsResult()
```

Comment:
_(sorry, I know this PR is still WIP but just thought I'd go ahead and post this suggestion :sweat_smile:)_

Can we add these assertions to [`test_hooks.test_pre_logout`](https://github.com/cal-itp/benefits/blob/main/tests/pytest/oauth/test_hooks.py#L52)?

Reply:
Not at all! This is great, I just pushed a small refactor of the `test_xxx_no_session_flow` tests but I'll look at this next 👍